tests/conftest.py

```diff
@@ -29,6 +30,19 @@ def pytest_make_parametrize_id(config, val, argname):
     return f"{val}"
 
 
+def pytest_ignore_collect(path: Path, config):
```

Comment:
I don't want to run `manual_test_examples.py` in CI because it's too slow now. But we are able to check it by 
```
pytest -s -v tests/manual_test_examples.py
```